ftl_installer/test_parser.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+from .config.Config import Config
+from .parser.Parser import Parser
```

Comment:
Should we move this test file to a `test` folder in the root of the project so that we can keep all of the tests in the same location? Also, tests should use full import paths for example `import ftl_installer.config.Config` so that we can mock them if needed.